### ISSUE TITLE
Zero (0) character output when rendering HelpControl

### ISSUE THREAD

**Issue by solepixel** [phpbits/block-options#221]
There's a bug somewhere with the HelpControl component where a zero (0) will output at the top of the page. Screenshot: [https://3cd.co/geur1eX4](https://3cd.co/geur1eX4)

The zero occurs inside the parent Fragment element, and just below the `.editorskit-component-help-tips` Dropdown component. I don't have much more information about the issue or how to reproduce it, but I sometimes see it appear when editing a Page.

This may or may not be related to having the [Stackable](https://wpstackable.com/) plugin installed, but it has been difficult to reproduce without this plugin activated, mainly due to the problem present on existing pages with complex content/layout.

Here is the `post_content` value directly from the database if it helps at all: https://3cd.co/7KuyXrO5

**Comment by phpbits** [comment]
@solepixel  Thanks a lot for reporting this issue. Does the "0" disappears when EditorsKit isn't activated? Thanks!

**Comment by solepixel** [comment]
Yes, the 0 goes away when EditorsKit is deactivated, however it also goes away when Stackable is deactivated, so it was hard to tell if the issue was a result of them both of them being active, however the output was contained within the EditorsKit element, which makes me feel like it's something coming from this plugin rather than Stackable.

**Comment by phpbits** [comment]
@solepixel Would you mind letting me know what version of Gutenberg are you using? I can't seem to replicate the issue. Thanks!
